sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3923,11 +3979,14 @@ SUBROUTINE MISCLN
                 ENDIF
                 STP=(EGRID1(I,J)/D1500)*SLCLtmp*(HELI(I,J,2)/150.)*&
                         FSHRtmp*SCINtmp
+                GRID1(I,J) = spval
+                IF(T1D(I,J) < spval) THEN
```

Comment:
remove this if block and add to the next if block, e.g.,
IF (STP>0 .and. STP<spval) THEN